crates/core/tedge_api/src/workflow/script.rs

```diff
@@ -241,7 +250,7 @@ impl ExitHandlers {
 
     pub fn forceful_timeout_extension(&self) -> Option<Duration> {
         self.timeout.map(|timeout| {
-            let extra = min(60, timeout.as_secs() / 20);
+            let extra = max(60, timeout.as_secs() / 20);
```

Comment:
Yeah this was buggy. We want to wait at least 1 minute before killing the process.